code/game/objects/items/devices/gps.dm

```diff
@@ -160,6 +160,13 @@ GLOBAL_LIST_EMPTY(GPS_list)
 	desc = "A mining cyborg internal positioning system. Used as a recovery beacon for damaged cyborg assets, or a collaboration tool for mining teams."
 	flags = NODROP
 
+/obj/item/device/gps/flightdatarecorder
+	name = "flight data recorder"
+	icon_state = "gps-fdr"
+	w_class = WEIGHT_CLASS_BULKY
+	gpstag = "Emergency Transponder - X"

```

Comment:
Aren't gps tags usually 4 letters?

Reply:
Nope, there is a character limit when a player sets it (20), but can display huge lines of text

![dreamseeker_2017-11-06_17-07-38](https://user-images.githubusercontent.com/29457447/32465993-554dcc24-c33d-11e7-8bf2-d0649a6ac232.jpg)

Reply:
Yeah but you should probably stick with the usual format of 4-letter tags

Reply:
I could get it down to 'BBET' but that's very vague. 'FDR-Emergency' is shorter and clearer as to what it is, if you are alright with that?

Reply:
Yeah, sure